logstash-core/lib/logstash/config/mixin.rb

```diff
@@ -147,7 +147,7 @@ def plugin_status(status = nil)
     # inside the gemspec.
     def milestone(m = nil)
       @logger = Cabin::Channel.get(LogStash)
-      @logger.warn(I18n.t('logstash.plugin.deprecated_milestone', :plugin => config_name))
+      @logger.debug(I18n.t('logstash.plugin.deprecated_milestone', :plugin => config_name))
```

Comment:
Hmm, I think we need this warning. Maybe at `info`. Question is, should this messages be logged when `pluginpath` option is used.